src/main/java/com/solvdLaba/airport/Main.java

```diff
@@ -14,8 +14,9 @@
 public class Main {
 
 
-    public static void main(String[] args) throws ParseException, InvalidBoardingPassException {
+    public static void main(String[] args) throws ParseException, InvalidBoardingPassException, NoSeatAvailable, InvalidAgeException {
```

Comment:
Main function should not throw exception. At this point you should already handle all the exceptions, so your main function always ends in the right place.

Reply:
I tried to use try-catch blocks to avoid adding exceptions in main function signature :)